fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/AccountNumberGenerator.java

```diff
@@ -45,6 +48,12 @@
     private static final String LOAN_PRODUCT_SHORT_NAME = "loanProductShortName";
     private static final String SAVINGS_PRODUCT_SHORT_NAME = "savingsProductShortName";
     private static final String SHARE_PRODUCT_SHORT_NAME = "sharesProductShortName";
+    private final ConfigurationReadPlatformService configurationReadPlatformService;
+
+    @Autowired
+    public AccountNumberGenerator(final ConfigurationReadPlatformService configurationReadPlatformService) {
+        this.configurationReadPlatformService = configurationReadPlatformService;
+    }
 
```

Comment:
I think these changes are from another PR that was already merged? Can you rebase your PR to develop?

Reply:
Done.